apps/showcase/src/services/webcontainer/webcontainer.service.ts

```diff
@@ -67,7 +70,9 @@ export class WebContainerService {
    */
   public async readFile(file: string): Promise<string> {
     const instance = await this.runner.instancePromise;
-
+    if (!(await this.isFile(file))) {
+      return '';
```

Comment:
shouldn't this be an exception ?

Reply:
I will rework this part, this was to avoid some edge case linked to switch of folder and async behaviour, but it will be more stable if I rework this part